diff -ru FabricUI.orig/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp FabricUI/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp	2016-01-13 13:54:58.000000000 -0500
+++ FabricUI/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp	2016-01-13 13:56:50.000000000 -0500
@@ -1502,7 +1502,7 @@
         if (!PyErr_Occurred()) {
             // getHelpText()const
             PyThreadState* _save = PyEval_SaveThread(); // Py_BEGIN_ALLOW_THREADS
-            char * cppResult = ((::DFGTabSearchWidgetWrapper*) cppSelf)->DFGTabSearchWidgetWrapper::getHelpText_protected();
+            const char * cppResult = ((::DFGTabSearchWidgetWrapper*) cppSelf)->DFGTabSearchWidgetWrapper::getHelpText_protected();
             PyEval_RestoreThread(_save); // Py_END_ALLOW_THREADS
             pyResult = Shiboken::Conversions::copyToPython(Shiboken::Conversions::PrimitiveTypeConverter<const char*>(), cppResult);
         }
diff -ru FabricUI.orig/fabricui_graphview_controller_wrapper.cpp FabricUI/fabricui_graphview_controller_wrapper.cpp
--- FabricUI.orig/fabricui_graphview_controller_wrapper.cpp	2016-01-13 13:54:58.000000000 -0500
+++ FabricUI/fabricui_graphview_controller_wrapper.cpp	2016-01-13 13:55:29.000000000 -0500
@@ -681,10 +681,10 @@
             void* addr = PySide::nextQObjectMemoryAddr();
             PyThreadState* _save = PyEval_SaveThread(); // Py_BEGIN_ALLOW_THREADS
             if (addr) {
-                cptr = new (addr) ::ControllerWrapper(cppArg0);
+                // cptr = new (addr) ::ControllerWrapper(cppArg0);
                 PySide::setNextQObjectMemoryAddr(0);
             } else {
-                cptr = new ::ControllerWrapper(cppArg0);
+                // cptr = new ::ControllerWrapper(cppArg0);
             }
 
             PyEval_RestoreThread(_save); // Py_END_ALLOW_THREADS
diff -ru FabricUI.orig/fabricui_graphview_graph_wrapper.cpp FabricUI/fabricui_graphview_graph_wrapper.cpp
--- FabricUI.orig/fabricui_graphview_graph_wrapper.cpp	2016-01-13 13:54:58.000000000 -0500
+++ FabricUI/fabricui_graphview_graph_wrapper.cpp	2016-01-13 13:55:29.000000000 -0500
@@ -1119,17 +1119,6 @@
     return cppResult;
 }
 
-const QMetaObject* GraphWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::GraphView::Graph::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int GraphWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::GraphView::Graph::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_graphview_node_wrapper.cpp FabricUI/fabricui_graphview_node_wrapper.cpp
--- FabricUI.orig/fabricui_graphview_node_wrapper.cpp	2016-01-13 13:54:58.000000000 -0500
+++ FabricUI/fabricui_graphview_node_wrapper.cpp	2016-01-13 13:55:29.000000000 -0500
@@ -1442,17 +1442,6 @@
     return cppResult;
 }
 
-const QMetaObject* NodeWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::GraphView::Node::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int NodeWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::GraphView::Node::qt_metacall(call, id, args);
