diff -ru FabricUI.orig/fabricui_dfg_dfgcontroller_wrapper.cpp FabricUI/fabricui_dfg_dfgcontroller_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfgcontroller_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_dfgcontroller_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -858,17 +858,6 @@
     return cppResult;
 }
 
-const QMetaObject* DFGControllerWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::DFGController::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int DFGControllerWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::DFGController::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_dfg_dfglogwidget_wrapper.cpp FabricUI/fabricui_dfg_dfglogwidget_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfglogwidget_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_dfglogwidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1251,17 +1251,6 @@
     return cppResult;
 }
 
-const QMetaObject* DFGLogWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::DFGLogWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int DFGLogWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::DFGLogWidget::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_dfg_dfgmainwindow_wrapper.cpp FabricUI/fabricui_dfg_dfgmainwindow_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfgmainwindow_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_dfgmainwindow_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1283,17 +1283,6 @@
     return cppResult;
 }
 
-const QMetaObject* DFGMainWindowWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::DFGMainWindow::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int DFGMainWindowWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::DFGMainWindow::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp FabricUI/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_dfgtabsearchwidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1303,17 +1303,6 @@
     return cppResult;
 }
 
-const QMetaObject* DFGTabSearchWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::DFGTabSearchWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int DFGTabSearchWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::DFGTabSearchWidget::qt_metacall(call, id, args);
@@ -1532,7 +1521,7 @@
         if (!PyErr_Occurred()) {
             // getHelpText()const
             PyThreadState* _save = PyEval_SaveThread(); // Py_BEGIN_ALLOW_THREADS
-            char * cppResult = ((::DFGTabSearchWidgetWrapper*) cppSelf)->DFGTabSearchWidgetWrapper::getHelpText_protected();
+            const char * cppResult = ((::DFGTabSearchWidgetWrapper*) cppSelf)->DFGTabSearchWidgetWrapper::getHelpText_protected();
             PyEval_RestoreThread(_save); // Py_END_ALLOW_THREADS
             pyResult = Shiboken::Conversions::copyToPython(Shiboken::Conversions::PrimitiveTypeConverter<const char*>(), cppResult);
         }
diff -ru FabricUI.orig/fabricui_dfg_dfgtabsearchwidget_wrapper.h FabricUI/fabricui_dfg_dfgtabsearchwidget_wrapper.h
--- FabricUI.orig/fabricui_dfg_dfgtabsearchwidget_wrapper.h	Wed Jan 13 15:18:18 2016
+++ FabricUI/fabricui_dfg_dfgtabsearchwidget_wrapper.h	Wed Jan 13 15:24:54 2016
@@ -50,7 +50,7 @@
     inline void focusOutEvent_protected(QFocusEvent * event) { FabricUI::DFG::DFGTabSearchWidget::focusOutEvent(event); }
     virtual void focusOutEvent(QFocusEvent * event);
     inline bool focusPreviousChild_protected() { return FabricUI::DFG::DFGTabSearchWidget::focusPreviousChild(); }
-    inline char * getHelpText_protected() const { return FabricUI::DFG::DFGTabSearchWidget::getHelpText(); }
+    inline const char * getHelpText_protected() const { return FabricUI::DFG::DFGTabSearchWidget::getHelpText(); }
     virtual int heightForWidth(int arg__1) const;
     inline int heightFromResults_protected() const { return FabricUI::DFG::DFGTabSearchWidget::heightFromResults(); }
     virtual void hideEvent(QHideEvent * event);
diff -ru FabricUI.orig/fabricui_dfg_dfgvalueeditor_wrapper.cpp FabricUI/fabricui_dfg_dfgvalueeditor_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfgvalueeditor_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_dfgvalueeditor_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1337,17 +1337,6 @@
     return cppResult;
 }
 
-const QMetaObject* DFGValueEditorWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::DFGValueEditor::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int DFGValueEditorWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::DFGValueEditor::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_dfg_dfgwidget_wrapper.cpp FabricUI/fabricui_dfg_dfgwidget_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_dfgwidget_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_dfgwidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1260,17 +1260,6 @@
     return cppResult;
 }
 
-const QMetaObject* DFGWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::DFGWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int DFGWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::DFGWidget::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_dfg_presettreewidget_wrapper.cpp FabricUI/fabricui_dfg_presettreewidget_wrapper.cpp
--- FabricUI.orig/fabricui_dfg_presettreewidget_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_dfg_presettreewidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1252,17 +1252,6 @@
     return cppResult;
 }
 
-const QMetaObject* PresetTreeWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::DFG::PresetTreeWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int PresetTreeWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::DFG::PresetTreeWidget::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_graphview_controller_wrapper.cpp FabricUI/fabricui_graphview_controller_wrapper.cpp
--- FabricUI.orig/fabricui_graphview_controller_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_graphview_controller_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -592,17 +592,6 @@
     return cppResult;
 }
 
-const QMetaObject* ControllerWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::GraphView::Controller::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int ControllerWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::GraphView::Controller::qt_metacall(call, id, args);
@@ -680,10 +669,10 @@
             void* addr = PySide::nextQObjectMemoryAddr();
             PyThreadState* _save = PyEval_SaveThread(); // Py_BEGIN_ALLOW_THREADS
             if (addr) {
-                cptr = new (addr) ::ControllerWrapper(cppArg0);
+                // cptr = new (addr) ::ControllerWrapper(cppArg0);
                 PySide::setNextQObjectMemoryAddr(0);
             } else {
-                cptr = new ::ControllerWrapper(cppArg0);
+                // cptr = new ::ControllerWrapper(cppArg0);
             }
 
             PyEval_RestoreThread(_save); // Py_END_ALLOW_THREADS
diff -ru FabricUI.orig/fabricui_graphview_graph_wrapper.cpp FabricUI/fabricui_graphview_graph_wrapper.cpp
--- FabricUI.orig/fabricui_graphview_graph_wrapper.cpp	Wed Jan 13 15:18:18 2016
+++ FabricUI/fabricui_graphview_graph_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1119,17 +1119,6 @@
     return cppResult;
 }
 
-const QMetaObject* GraphWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::GraphView::Graph::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int GraphWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::GraphView::Graph::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_graphview_node_wrapper.cpp FabricUI/fabricui_graphview_node_wrapper.cpp
--- FabricUI.orig/fabricui_graphview_node_wrapper.cpp	Wed Jan 13 15:18:18 2016
+++ FabricUI/fabricui_graphview_node_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1442,17 +1442,6 @@
     return cppResult;
 }
 
-const QMetaObject* NodeWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::GraphView::Node::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int NodeWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::GraphView::Node::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_style_fabricstyle_wrapper.cpp FabricUI/fabricui_style_fabricstyle_wrapper.cpp
--- FabricUI.orig/fabricui_style_fabricstyle_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_style_fabricstyle_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -837,17 +837,6 @@
     }
 }
 
-const QMetaObject* FabricStyleWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::Style::FabricStyle::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int FabricStyleWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::Style::FabricStyle::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_valueeditor_valueeditorwidget_wrapper.cpp FabricUI/fabricui_valueeditor_valueeditorwidget_wrapper.cpp
--- FabricUI.orig/fabricui_valueeditor_valueeditorwidget_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_valueeditor_valueeditorwidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1331,17 +1331,6 @@
     return cppResult;
 }
 
-const QMetaObject* ValueEditorWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::ValueEditor::ValueEditorWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int ValueEditorWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::ValueEditor::ValueEditorWidget::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_viewports_glviewportwidget_wrapper.cpp FabricUI/fabricui_viewports_glviewportwidget_wrapper.cpp
--- FabricUI.orig/fabricui_viewports_glviewportwidget_wrapper.cpp	Wed Jan 13 15:18:18 2016
+++ FabricUI/fabricui_viewports_glviewportwidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1472,17 +1472,6 @@
     return cppResult;
 }
 
-const QMetaObject* GLViewportWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::Viewports::GLViewportWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int GLViewportWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::Viewports::GLViewportWidget::qt_metacall(call, id, args);
diff -ru FabricUI.orig/fabricui_viewports_timelinewidget_wrapper.cpp FabricUI/fabricui_viewports_timelinewidget_wrapper.cpp
--- FabricUI.orig/fabricui_viewports_timelinewidget_wrapper.cpp	Wed Jan 13 15:18:17 2016
+++ FabricUI/fabricui_viewports_timelinewidget_wrapper.cpp	Wed Jan 13 15:24:54 2016
@@ -1251,17 +1251,6 @@
     return cppResult;
 }
 
-const QMetaObject* TimeLineWidgetWrapper::metaObject() const
-{
-    #if QT_VERSION >= 0x040700
-    if (QObject::d_ptr->metaObject) return QObject::d_ptr->metaObject;
-    #endif
-    SbkObject* pySelf = Shiboken::BindingManager::instance().retrieveWrapper(this);
-    if (pySelf == NULL)
-        return FabricUI::Viewports::TimeLineWidget::metaObject();
-    return PySide::SignalManager::retriveMetaObject(reinterpret_cast<PyObject*>(pySelf));
-}
-
 int TimeLineWidgetWrapper::qt_metacall(QMetaObject::Call call, int id, void** args)
 {
     int result = FabricUI::Viewports::TimeLineWidget::qt_metacall(call, id, args);
